hypervisor/arch/x86/Kconfig

```diff
@@ -1,92 +1,141 @@
 choice
 	prompt "Platform"
 	default PLATFORM_SBL
+	help
+	  The BIOS used on the target board.
 
 config PLATFORM_UEFI
 	bool "UEFI"
 	select EFI_STUB
+	help
+	  Select this if the target board uses UEFI.
 
 config PLATFORM_SBL
 	bool "SBL"
+	help
+	  Select this if the target board uses Slim Bootloader.
 
 endchoice
 
 choice
 	prompt "Hypervisor mode"
 	default SHARING_MODE
+	help
+	  Select the usage mode of the hypervisor.
 
 config SHARING_MODE
 	bool "Sharing mode"
+	help
+	  In sharing mode, the first VM (called Service OS or SOS) is capable of
+	  managing the other VMs (called User OS or UOS) and trap/mediate their
+	  I/O accesses to physical devices.
 
 config PARTITION_MODE
 	bool "Partition mode"
 	depends on PLATFORM_SBL
+	help
+	  In partition mode, every VM owns part of the physical resources
+	  exclusively and runs with minimal interference from the others. The VM
+	  configurations are statically defined in the source.
+
 endchoice
 
 config PLATFORM
 	string
 	default "uefi" if PLATFORM_UEFI
 	default "sbl" if PLATFORM_SBL
 
+config BOARD
+	string "Target board"
+	help
+	The target board this build runs on top of.
+
+config DEFCONFIG_LIST
+	string
+	option defconfig_list
+	default "arch/x86/configs/$BOARD.config"
```

Comment:
The BOARD is an environment variable passed to silentoldconfig.py, which is guaranteed to exist when building the hypervisor. When building at the top level directory, the build is for apl-nuc because PLATFORM=uefi is given. When building in the hypervisor/ directory, the board is apl-mrb if BOARD is not given in the command line.